packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimePickerFooter.tsx

```diff
@@ -43,42 +53,85 @@ export const TimePickerFooter: FC<Props> = (props) => {
     return null;
   }
 
-  if (isEditing) {
-    return (
-      <div className={cx(style.container, style.editContainer)}>
-        <section aria-label={selectors.components.TimeZonePicker.container} className={style.timeZoneContainer}>
-          <TimeZonePicker
-            includeInternal={true}
-            onChange={(timeZone) => {
-              onToggleChangeTz();
-
-              if (isString(timeZone)) {
-                onChangeTimeZone(timeZone);
-              }
-            }}
-            autoFocus={true}
-            onBlur={onToggleChangeTz}
-          />
-        </section>
-      </div>
-    );
-  }
-
   return (
-    <section aria-label="Time zone selection" className={style.container}>
-      <div className={style.timeZoneContainer}>
-        <div className={style.timeZone}>
-          <TimeZoneTitle title={info.name} />
-          <div className={style.spacer} />
-          <TimeZoneDescription info={info} />
+    <div>
+      <section aria-label="Time zone selection" className={style.container}>
+        <div className={style.timeZoneContainer}>
+          <div className={style.timeZone}>
+            <TimeZoneTitle title={info.name} />
+            <div className={style.spacer} />
+            <TimeZoneDescription info={info} />
```

Comment:
I'm missing information about what's the currently selected fiscal year start month here. I know it's not in the designs, yet I think we should be slightly more verbose about what the selection is, similar to what we do with a time zone. Otherwise the fiscal year setting is super tricky to discover if you don't know where to look for it. What do you think @amy-super?

![image](https://user-images.githubusercontent.com/2376619/135077891-13134530-a062-4ec7-a237-dcfed0da6813.png)